chessence-backend/models/user-services.js

```diff
@@ -121,9 +121,25 @@ async function updateProfile(id, user, oldPwd) {
     // }
 }
 
+async function getUsernameById(userId) {
+    //try {
+    const user = await userModel.findById(userId);
+
+    if (!user) {
+        return null;
+    }
+
+    return user.username;
+    // } catch (error) {
```

Comment:
why is this try catch block commented out?

Reply:
Because it is gonna fail the coverage test.